tapir/coop/migrations/0039_draftuser_share_owner.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.23 on 2023-12-05 14:16
```

Comment:
Out of interest, how are these generated?

Reply:
With the `makemigrations` command. Django then checks for changes in the models and generates the corresponding migrations, we never write them manually.
You also have to apply them with `migrate`.
```sh
docker compose exec web poetry run python manage.py makemigrations
```